Sources/WebSocket/WebSocket.swift

```diff
@@ -73,8 +82,18 @@ public final class WebSocket: BasicWorker {
         self.onBinaryCallback = { _, _ in }
         self.onErrorCallback = { _, _ in }
         self.onCloseCodeCallback = { _ in }
+        
+        self.onTextCallback = { _, text in
```

Comment:
You're reassigning the onTextCallback/onBinaryCallback/onErrorCallback. I believe you intended to remove the assignments above?